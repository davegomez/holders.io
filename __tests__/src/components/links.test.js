import React from 'react'
import Links from '../../../src/components/links'
import renderer from 'react-test-renderer'

test('Links component', () => {
  const component = renderer.create(
    <Links url={{ pathname: '/' }}>Facebook</Links>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
