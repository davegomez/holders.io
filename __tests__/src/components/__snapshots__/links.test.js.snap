// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links component 1`] = `
<ul
  data-jsx={1218288577}
>
  <li
    data-jsx={1218288577}
  >
    Home
  </li>
  <li
    className="active"
    data-jsx={1218288577}
  >
    <a
      data-jsx={1218288577}
      href="/docs"
      onClick={[Function]}
    >
      Documentation
    </a>
  </li>
  <li
    className="active"
    data-jsx={1218288577}
  >
    <a
      data-jsx={1218288577}
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
  </li>
</ul>
`;
